### ISSUE TITLE
NFT Transfer should inlcude its direction

### ISSUE THREAD

**Issue by Keith-CY** [Magickbase/ckb-explorer-public-issues#503]
E.g. https://ckb-explorer-api.staging.layerview.io//api/v2/ckb_transactions/0xc7d1eae3dfe4b8a675b5343592f0e81e6b2e3d82275e57bf883874597ea85aa2/details

```json
{
  "data": [
    {
      "address": "ckt1qrejnmlar3r452tcg57gvq8patctcgy8acync0hxfnyka35ywafvkqgpr2dgu6fydzt2c9eqc86qwvyuzsk9c0xuqqn68m6w",
      "transfers": [
        {
          "capacity": "-2109500000000.0",
          "cell_type": "spore_cell"
        }
      ]
    },
    {
      "address": "ckt1qrejnmlar3r452tcg57gvq8patctcgy8acync0hxfnyka35ywafvkqgpes90ptu3rh2qs5acer07ay9n97x3ajkkqqnm062a",
      "transfers": [
        {
          "capacity": "2109499978498.0",
          "cell_type": "spore_cell"
        }
      ]
    }
  ]
}
```

It cannot be deduced that `spore` is transferred from `Address 1` to `Address 2` because capacity could be increased with a sending out of NFT.

Imagine the case: in a transaction, User A sends more capacity to B and B sends an NFT to User A, it's a typical atomic transaction for buying an NFT. The result is the same as the one above.

**Comment by Keith-CY** [comment]
A similar case can be found at https://ckb-explorer-frontend-in-magickbase-repo-git-7edfd6-magickbase.vercel.app/transaction/0x04ecbc2df39e3682326a3b23c1bd2465e07eae2379ac0cc713834a1f79753779?layout=lite
notice address `ckt1qpwvfjzparx0pv8mxv6v0yp7wtdrn8pl4xq4cpaed3ffwwxrspe4xqvfxv3m64sed7zjkzef9gw74p65cup6sx44furc54jvxr2p2p6g6uf6vguq` got CKB with sending sudt

**Comment by Sven-TBD** [comment]
https://vs0cjf.axshare.com/?id=rum8j6&g=14&p=lite_%E6%A0%B7%E5%BC%8F%E4%B8%ADa_b_%E6%A0%BC%E5%BC%8F%E8%B0%83%E6%95%B4%E8%AF%B4%E6%98%8E

TLDR,
Supplementary Rule on the color of Amount text field.
In the data display format of A(B), the contents of parts A and B will be assigned corresponding colors based on whether the A and B type assets are actually gained or lost in a specific transaction.

**Comment by Keith-CY** [comment]
> [vs0cjf.axshare.com/?id=rum8j6&g=14&p=lite_%E6%A0%B7%E5%BC%8F%E4%B8%ADa_b_%E6%A0%BC%E5%BC%8F%E8%B0%83%E6%95%B4%E8%AF%B4%E6%98%8E](https://vs0cjf.axshare.com/?id=rum8j6&g=14&p=lite_%E6%A0%B7%E5%BC%8F%E4%B8%ADa_b_%E6%A0%BC%E5%BC%8F%E8%B0%83%E6%95%B4%E8%AF%B4%E6%98%8E)
> 
> TLDR, Supplementary Rule on the color of Amount text field. In the data display format of A(B), the contents of parts A and B will be assigned corresponding colors based on whether the A and B type assets are actually gained or lost in a specific transaction.

Color of parentheses should be fixed by https://github.com/Magickbase/ckb-explorer-frontend/pull/183

**Comment by rabbitz** [comment]
https://github.com/nervosnetwork/ckb-explorer/pull/1565

**Comment by Keith-CY** [comment]
Work on UI will start when the update is deployed onto the staging

**Comment by rabbitz** [comment]
> Work on UI will start when the update is deployed onto the staging

The update has already been deployed onto the staging.

**Comment by rabbitz** [comment]
https://github.com/nervosnetwork/ckb-explorer/pull/1625
